source/includes/_accepting-payments.md

```diff
@@ -316,7 +316,7 @@ Below are the things that you can customize:
 
 2. Admin fee (choose whether the admin fee will be borne by your customers or borne by you)
 
-3. Payment method (choose the payment methods displayed to your customers among Bank Transfer (via Virtual Account), Cards (Credit Card/Debit Card), E-Wallet (ShopeePay), and QR Code options. Additionally, you can choose which banks are enabled for Bank Transfer method.)
+3. Payment method (choose the payment methods displayed to your customers among Bank Transfer (via Virtual Account), Cards (Credit Card/Debit Card), E-Wallet (ShopeePay, DANA, LinkAja, OVO), and QR Code options. Additionally, you can choose which banks are enabled for Bank Transfer method.)
```

Comment:
ini sudah live @andiniekaf22 ? apakah emang secara sengaja mau kita publish sebelom kita live ?

Reply:
Sengaja Do, utk tarik partner yg interested buat pake (part of GTM strategy). Di Landing Page juga sama approachnya (munculin logo2nya sebelum live)